src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -741,9 +741,28 @@ get_xrt_pretty_version()
   for(auto& drv : available_drivers) {
     const boost::property_tree::ptree& driver = drv.second;
     std::string drv_name = driver.get<std::string>("name", "N/A");
-    boost::algorithm::to_upper(drv_name);
-    ss << boost::format("%-20s : %s, %s\n") % drv_name
-        % driver.get<std::string>("version", "N/A") % driver.get<std::string>("hash", "N/A");
+    std::string drv_hash = driver.get<std::string>("hash", "N/A");
```

Comment:
Is this code coming from somewhere? Can we make it common code so that we don't have to change in 2 places if there are any changes in the future

Reply:
This is coming from Report Host : ReportHost::writeReport. The ptree used in ReportHost is different from what's being used here as I understand. The ReportHost ptree queries using "host.*" strings. Not sure if this can be re-used.

Reply:
Thanks for pointing me to the code! I think it can be re-used. Define a `print_xrt_info(boost::property_tree::ptree& driver_ptree)` in XBUtilites.cpp and call it in Host Report as `print_xrt_info(_pt.get_child("host.xrt.drivers", empty_ptree))`

Reply:
Good Idea. It's done now